src/doc/markdown.lisp

```diff
@@ -226,10 +226,15 @@
                  :regex "[<>&]")))
 
       (format stream "Methods:~%")
-      (loop :for (name . type) :in methods :do
-        (format stream "- <code>~A :: ~A</code>~%"
+      (loop :for (name . type) :in methods
+            :for docstring     :in method-docstrings :do
+
+        (format stream "- <code>~A :: ~A</code>~a~%"
                 (html-entities:encode-entities (symbol-name name))
-                (to-markdown type))))
+                (to-markdown type)
+                (if docstring
```

Comment:
use the format directive

    ~@[<br/>~A~]

in the surrounding FORMAT.

This will skip the thing in brackets if the argument is nil, otherwise it'll run the thing in brackets with the argument supplied to the next tilde directive.

Allows you to eliminate the IF and nested FORMAT.